run_scripts/2_fetch_raw_data.sh

```diff
@@ -14,11 +14,12 @@ RP_SERVER=$3
 RP_TOKEN=$4
 DATA_ROOT=$5
 
-TEST_CONTACTS_PATH="$(pwd)/data/test_contacts.json"
+TEST_CONTACTS_PATH="$(pwd)/configurations/test_contacts.json"
```

Comment:
It looks like you're including some fixes relevant to the previous PR on this branch.

If a PR is approved with comments, better to address those on that PR rather than the next one. Small changes like this don't need re-review.

Reply:
Had already pushed this branch awaiting previous branch review so had to make changes to align to the comments.